schema.go

```diff
@@ -217,6 +217,22 @@ var Schema = schema.NewTypedScopeSchema[*Config](
 				schema.PointerTo(util.JSONEncode(string(ImagePullPolicyIfNotPresent))),
 				nil,
 			),
+			"startDelayMS": schema.NewPropertySchema(
+				schema.NewIntSchema(schema.PointerTo(int64(0)), schema.PointerTo(int64(10000)), nil),
```

Comment:
Is the maximum required?  10s seems like an arbitrary value, and an arbitrarily low value, at that.  Since the maximum is not the default, would it hurt to pick something outlandishly high, just so that this doesn't unnecessarily constrain the user?  (Values like `4294967295`, `2147483648`, `2147483647` and `999999999` leap to mind, but there are other good ones....)

Reply:
It's not required. But this is meant to be used for workarounds. When I used this for a workaround, 500ms was enough. This 20 times that, so I think that's a large enough for any case.
This should only be used if absolutely necessary. Most users shouldn't need to use this.
Do you think that's okay?

Reply:
If the maximum is not required, then I recommend omitting it and accepting any (positive) value (up to the limit of an `int64`...).

I understand that we hope that no one will _have_ to use this, but if we're going to go to the trouble of providing it, why should there be an upper limit on the delay?  What does it cost Arca, or what does it force the implementation to do, that limiting the delay to under 10s saves us from?  (Why is 20x the right number?)

Some user might find a novel use for this feature.  Why should we limit it?  (If there is a good reason, then I'm fine with that; however, if there is no reason, then lets not do it.)

Reply:
Updated.